Library/Formula/fmpp.rb

```diff
@@ -0,0 +1,21 @@
+class Fmpp < Formula
+  desc "FMPP is a text file preprocessing tool using FreeMarker templates."
+  homepage "http://fmpp.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/fmpp/fmpp/0.9.15/fmpp_0.9.15.tar.gz"
+  sha256 "b893451b5450a7f35fe680e934f6903ec8143d88959dcfca5d17467fbe4142f9"
+  head "https://github.com/freemarker/fmpp.git"
+
+  depends_on :java
```

Comment:
I'm not sure it's worth specifying a Java minimum dependency that early; you'd have to go out of your way to install Java 1.4 these days. Thoughts @xu-cheng?